cplusplus/include/vips/VInterpolate8.h

```diff
@@ -49,7 +43,7 @@ class VInterpolate : VObject
 	}
 
 	static 
-	VInterpolate new_from_name( const char *name, VOption *options = 0 );
+	VInterpolate new_from_name( const char *name );
```

Comment:
Should I add a compat stub for `VInterpolate::new_from_name` preserving the `VOption *` parameter? I think this change isn't ABI compatible:
https://kleisauke.nl/compat_reports/libvips-cpp/8.8.3_to_8.9.0/compat_report.html

The `libvips.so` seems to be fine:
https://kleisauke.nl/compat_reports/libvips/8.8.3_to_8.9.0/compat_report.html

(only internal API symbols were removed)

Reply:
Yes, I was wondering about that.

Although the options aren't used right now, `VipsInterpolate` was originally designed to support parameterised interpolators and they might come back. If they do, we'd need to re-add the options param and it would break ABI again.

I'd be inclined to revert this change, unless there is a good reason.

Reply:
Ah, I didn't know that `VipsInterpolate` was originally designed to support parameterised interpolators. Let's revert this change!

Reply:
OK, reverted with https://github.com/libvips/libvips/commit/92521c0dc6ac5d4078107e35f5ea16c96ac2cbf8